packages/generate/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gel/generate",
-  "version": "0.5.6",
+  "version": "1.0.0",
```

Comment:
I think we should consider 0.6.0 here: there are a number of other breaking changes we might want to make before really settling on 1.0 which we will try to keep around for a long time without breaking.

Reply:
yes ofc, I'm fine with 0.6.0